tests/localosmosis/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+
+services:
+  osmosisd:
```

Comment:
You could add:

```
osmosisd:
  build:
      context: .
```

In this way when you run `docker-compose start` it would automatically build the image for you! :)